pkg/skaffold/build/local/local.go

```diff
@@ -151,3 +144,16 @@ func (b *Builder) getImageIDForTag(ctx context.Context, tag string) (string, err
 	}
 	return insp.ID, nil
 }
+
+func (b *Builder) SyncMap(ctx context.Context, a *latest.Artifact) (map[string][]string, error) {
+	if a.DockerArtifact == nil {
+		return nil, build.ErrSyncMapNotSupported{}
+	}
+
+	syncMap, err := docker.SyncMap(ctx, a.Workspace, a.DockerArtifact.DockerfilePath, a.DockerArtifact.BuildArgs, b.insecureRegistries)
```

Comment:
can we not just return `docker.SyncMap(....)` here? I think it's more idiomatic to check `if err != nil` in the caller and then conditionally use the returned `syncMap`.

Reply:
oh yeah, sure! David made me pull other code out from there, but I missed that this can be further simplified..